book/website/reproducible-research/licensing/licensing-hardware.md

```diff
@@ -0,0 +1,136 @@
+(rr-licensing-hardware)=
+# Open Hardware Licenses
+
+(rr-licensing-hardware-prerequisites)=
+## Prerequisites
+
+This chapter builds on the chapter {ref}`rr-licensing`.
+It is advisable to also read the chapter {ref}`rr-licensing-software` before or after reading this one.
+
+## Introduction
+
+While this chapter focuses on open source hardware licenses, this should be considered within the broader open source movement. 
+The open source licensing of hardware came out of open source software licenses and, in fact, several open source software licenses are widely accepted within the open hardware community. 
+Please refer to the warnings and description of terms you will find in the introductory chapter
+{ref}`rr-licensing`.
+
+As explained there, for many open source licenses to be applicable, it is necessary to figure out what parts of the project are actually copyrightable and therefore can be licensed.
+This complicates thing for hardware, as copyrights do not protect functional items and hardware can be a combination of creative and functional elements.
+While the former is protected by copyright, the latter would usually need a patent (if it is indeed patentable) to be protected.
+It is important to note that copyrights and patents are mutually exclusive: something is either protected by copyright or it is protected by a patent, but not both simultaneously.
+
+
+
+### What licenses to use
```

Comment:
I believe the wording here might be 'Which' licenses to use!

Reply:
```suggestion
### Which licenses to use
```